pkg/controllers/securitypolicy/securitypolicy_controller.go

```diff
@@ -23,40 +23,31 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/event"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
 	"github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1"
+	"github.com/vmware-tanzu/nsx-operator/pkg/controllers"
 	"github.com/vmware-tanzu/nsx-operator/pkg/metrics"
 	_ "github.com/vmware-tanzu/nsx-operator/pkg/nsx/ratelimiter"
-	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services"
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/securitypolicy"
 	util2 "github.com/vmware-tanzu/nsx-operator/pkg/nsx/util"
 	"github.com/vmware-tanzu/nsx-operator/pkg/util"
 )
 
-const (
-	METRIC_RES_TYPE = "securitypolicy"
-)
-
-var (
-	log                     = logf.Log.WithName("controller").WithName("securitypolicy")
-	resultNormal            = ctrl.Result{}
-	resultRequeue           = ctrl.Result{Requeue: true}
-	resultRequeueAfter5mins = ctrl.Result{Requeue: true, RequeueAfter: 5 * time.Minute}
-)
+var log = controllers.Log
 
-// SecurityPolicyReconciler SecurityPolicyReconcile reconciles a SecurityPolicy object
-type SecurityPolicyReconciler struct {
+// Reconciler SecurityPolicyReconcile reconciles a SecurityPolicy object
```

Comment:
At this momment, our target it re-org code in directory and structural level. It's unnecessary to change much on code level. So,I suggest that we still keep SecurityPolicyReconciler naming for this SecurityPolicy controoler. 
Also, we need to take into account of code maintenance on all released branches since SecurityPolicy is a released feature other than a new one.  We had better to minimized the code change for this feature across branches. Otherwise, it bring more efforts for us to maintain code.

Reply:
ok